src/SourceBuild/content/test/Microsoft.DotNet.Installer.Tests/LinuxInstallerTests.cs

```diff
@@ -55,7 +55,9 @@ public LinuxInstallerTests(ITestOutputHelper outputHelper)
         _contextDir = Path.Combine(_tmpDir, Path.GetRandomFileName());
         Directory.CreateDirectory(_contextDir);
 
-        _excludeLinuxArch = Config.Architecture == Architecture.X64 ? "arm64" : "x64";
+        _excludeLinuxArch = Config.Architecture == Architecture.X64 ?
```

Comment:
Could this just be the following?  Why is the condition necessary?
```suggestion
        _excludeLinuxArch = Architecture.Arm64.ToString().ToLower() 
```

Reply:
Wrong example... Config.Architecture.ToString().ToLower()

Reply:
For `x64` tests we exclude `arm64` packages from copying to the image. For `arm64` tests we exclude `x64` packages.

Reply:
I see - taking a step back, my question is what will happen if/when additional architectures are added?  Ideally this code would be written in a way that would be unaffected.

Reply:
If we add another supported architecture for testing, we'd need to modify this code and the `Architecture` enum, at the same time.

This member field as an optimization only. We could just iterate over all supported architectures that are not being tested, in the filtering method. That would incur performance cost.

Ideally, as we discussed before, I think filtering should be modified to consume the merged manifest and filter packages based on vertical name/Id. I can do a follow up PR with that change.

Reply:
>I think filtering should be modified to consume the merged manifest and filter packages based on vertical name/Id. I can do a follow up PR with that change.

Sounds good to me.